### ISSUE TITLE
how to upload pdf?

### ISSUE THREAD

**Issue by chebyte** [TeamOneTickets/VividSeats#1]
hi guys I was seeing your api and I have a question, I saw how to send the ticket information but I cant figure out how can I to do for upload the pdf?

thanks for advance

**Comment by jwcobb** [comment]
I don't think that functionality was ever built in the library. I'm happy to take a PR if you want to add it though.

I'm also happy to let someone else take over this project as I am no longer a ticket broker and no longer use this.

**Comment by chebyte** [comment]
sure I could add this new feature, but the problem is that in the doc api https://brokers.vividseats.com/ListingWebServices_v1.pdf
 I dont see any way to upload the pdf

do you know the workflow?

thanks

**Comment by jwcobb** [comment]
It appears to be a separate thing (fulfillment vs listing): https://brokers.vividseats.com/FulfillmentWebServices_v1.pdf
